packages/frontend/components/Sidebar/SectionRequirement.tsx

```diff
@@ -47,6 +49,23 @@ const SectionRequirement: React.FC<SidebarRequirementProps> = ({
     }
   };
 
+  const isCourseInPlan = (requirement: Requirement2): boolean => {
+    let isTrue = false;
+    if (coursesTaken) {
+      coursesTaken.forEach((course) => {
+        if (
+          requirement.type === "COURSE" &&
+          course.classId === requirement.classId.toString()
```

Comment:
courseID is like 2500, so if CS2500 is added, it would say another class like GAME2500 is added
![image](https://github.com/user-attachments/assets/e36a49ac-846c-458d-8488-4d2500974822)